draft-ietf-quic-transport.md

```diff
@@ -2816,6 +2960,10 @@ ACK Block (repeated):
 
 ### Sending ACK Frames
 
+ACK Frames MAY be sent when all packets to be acknowledge had an IP header with
```

Comment:
acknowledged

Reply:
Also, I think leading with this paragraph here is a bit confusing.  I would prefer not adding anything here about ACK_ECN and placing text in the ACK_ECN section that must be used when new marks are received.

Reply:
Okay, I removed this paragraph, and moved the MAY statement to the ACK_ECN frame, the MUST statement was already present.